R/class-yproject.R

```diff
@@ -181,16 +182,20 @@ ys_project <- function(..., output=tempfile(fileext=".yml"),
     dups <- paste0("  ", dups)
     stop("Duplicated spec names:\n", paste0(dups,collapse = "\n"))
   }
-  output <- normalPath(output,mustWork=FALSE)
+  output <- normalPath(output, mustWork = FALSE)
   if(missing(sponsor)) {
     if(.has("sponsor", meta)) {
-      sponsor <- meta$sponsor[1]  
+      if(nchar(meta[["sponsor"]][[1]]) > 0) {
```

Comment:
@kylebaron 

The check for nchar uses double brackets around the `[[1]]` (line 188), but when it gets assigned single brackets are used (line 189). Is this intended?

Same is true for lines 195 and 196

Reply:
Thanks, @andersone1 ; I pushed a change to we consistently use double; double and single work the same on vectors but makes sense to get it conssitent.